config/config.default.yaml

```diff
@@ -991,6 +996,150 @@ adjustments:
+data:
+  hotmaps_industrial_sites:
+    source: "archive"
+    version: "latest"
+  enspreso_biomass:
+    source: "archive"
+    version: "latest"
+  osm:
+    source: "archive"
+    version: "latest"
+  worldbank_urban_population:
+    source: "archive"
+    version: "latest"
+  gem_europe_gas_tracker:
+    source: "archive"
+    version: "latest"
+  co2stop:
+    source: "archive"
+    version: "latest"
+  nitrogen_statistics:
+    source: "archive"
+    version: "latest"
+  eu_nuts2013:
+    source: "archive"
+    version: "latest"
+  eu_nuts2021:
+    source: "archive"
+    version: "latest"
+  eurostat_balances:
+    source: "archive"
+    version: "latest"
+  eurostat_household_balances:
+    source: "archive"
+    version: "latest"
+  wdpa:
+    source: "archive"
+    version: "latest"
+  wdpa_marine:
+    source: "archive"
+    version: "latest"
+  luisa_land_cover:
+    source: "archive"
+    version: "latest"
+  jrc_idees:
+    source: "archive"
+    version: "latest"
+  scigrid_gas:
+    source: "primary"
+    version: "latest"
+  synthetic_electricity_demand:
+    source: "primary"
+    version: "latest"
+  copernicus_land_cover:
+    source: "primary"
+    version: "latest"
+  ship_raster:
+    source: "archive"
+    version: "latest"
+  eez:
+    source: "archive"
+    version: "latest"
+  nuts3_population:
+    source: "archive"
+    version: "latest"
+  gdp_per_capita:
+    source: "archive"
+    version: "latest"
+  population_count:
+    source: "archive"
+    version: "latest"
+  ghg_emissions:
+    source: "archive"
+    version: "latest"
+  gebco:
+    source: "archive"
+    version: "latest"
+  attributed_ports:
+    source: "archive"
+    version: "latest"
+  corine:
+    source: "archive"
+    version: "latest"
+  emobility:
+    source: "archive"
+    version: "latest"
+  h2_salt_caverns:
+    source: "archive"
+    version: "latest"
+  lau_regions:
+    source: "archive"
+    version: "latest"
+  aquifer_data:
+    source: "archive"
+    version: "latest"
+  osm_boundaries:
+    source: "archive"
+    version: "latest"
+  gem_gspt:
+    source: "archive"
+    version: "latest"
+  tyndp:
+    source: "archive"
+    version: "latest"
+  powerplants:
+    source: "primary"
+    version: "latest"
+  costs:
+    source: "primary"
+    version: "latest"
+  country_runoff:
+    source: "archive"
+    version: "latest"
+  country_hdd:
+    source: "archive"
+    version: "latest"
+  natura:
+    source: "archive"
+    version: "latest"
+  bfs_road_vehicle_stock:
+    source: "primary"
+    version: "latest"
+  bfs_gdp_and_population:
+    source: "primary"
+    version: "latest"
+  mobility_profiles:
+    source: "archive"
+    version: "latest"
+  cutout:
+    source: "archive"
+    version: "latest"
+  dh_areas:
+    source: "archive"
+    version: "latest"
+  geothermal_heat_utilisation_potentials:
+    source: "archive"
+    version: "latest"
+  jrc_ardeco:
+    source: "archive"
+    version: "latest"
+
+secrets:
```

Comment:
Great, this section will also be introduced in https://github.com/PyPSA/pypsa-eur/pull/1828 for ENTSO-E API access via `entsoepy`.